netbox.go

```diff
@@ -69,14 +70,12 @@ func (n *Netbox) ReadFromNetbox(ctx context.Context, Host string, ValidationToke
 	//Get the Interfaces list from netbox to populate the Machine gateway and nameserver value
 	ipamReq := ipam.NewIpamIPRangesListParams()
 	n.ReadIpRangeFromNetbox(ctx, c, ipamReq)
-	if n.debug {
-		n.logger.Info("ALL DEVICES")
-	}
+
+	n.logger.V(1).Info("ALL DEVICES")
 
 	for _, machine := range n.Records {
-		if n.debug {
-			n.logger.Info(machine.Hostname, machine.IPAddress, machine.MACAddress, machine.BMCIPAddress)
-		}
+		n.logger.V(1).Info(machine.Hostname, machine.IPAddress, machine.MACAddress, machine.BMCIPAddress)
```

Comment:
This looks like this is missing a message. the func signature sure that a `msg` then `key,value` pairs.